### DISCUSSION TITLE
问题

### DISCUSSION THREAD

**Discussion by ghost** [wangxinleo/wechat-public-account-push#184] [Show and tell]
大哥，这个啥问题呀
![image](https://user-images.githubusercontent.com/113239963/190897719-1be5265b-8f0b-48ad-b10b-57c0ef0e2724.png)

**Comment by wangxinleo** [comment]
> 大哥，这个啥问题呀
> ![image](https://user-images.githubusercontent.com/113239963/190897719-1be5265b-8f0b-48ad-b10b-57c0ef0e2724.png)
> 

你看错脚本了。不是这个脚本，是那个wechat-push-on-time那个！

**Comment by Diamondunn** [comment]
<img width="992" alt="1663950945603" src="https://user-images.githubusercontent.com/114246922/192009632-bbd449b5-0658-499b-85d4-65e1a823debc.png">
这个是因为什么问题

**Reply by wangxinleo** [reply]
不要使用中文的冒号，编辑的时候把翻译关掉，

**Reply by Diamondunn** [reply]
并没有开翻译，也没使用中文冒号还是这个报错

**Reply by Diamondunn** [reply]
<img width="1280" alt="1664003953403" src="https://user-images.githubusercontent.com/114246922/192085508-836a3a44-5c4d-470d-9c31-0a0ace47137f.png">

**Comment by wangxinleo** [comment]
难道不是吗？ 报错信息就指着那个冒号是中文 : 和：的区别


从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年9月24日 15:16
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [wangxinleo/wechat-public-account-push] 问题 (Discussion #184)


并没有开翻译，也没使用中文冒号还是这个报错

―

**Comment by wangxinleo** [comment]
要不帮忙把网址发过来，我帮你看看配置文件


从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年9月24日 15:19
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [wangxinleo/wechat-public-account-push] 问题 (Discussion #184)


[1664003953403]<https://user-images.githubusercontent.com/114246922/192085508-836a3a44-5c4d-470d-9c31-0a0ace47137f.png>

―

**Reply by Diamondunn** [reply]
https://github.com/Diamondunn/wechat-public-account-push

**Comment by wangxinleo** [comment]
我知道了~这个脚本跟你没关系噢…………这个是test脚本，你看到关键字了嘛 npm run test~~ 你不需要关注这个脚本，你应该关注其他脚本…………

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Diamondunn ***@***.***>
发送时间: Saturday, September 24, 2022 3:16:03 PM
收件人: wangxinleo/wechat-public-account-push ***@***.***>
抄送: wangxin.leo ***@***.***>; Comment ***@***.***>
主题: Re: [wangxinleo/wechat-public-account-push] 问题 (Discussion #184)


并没有开翻译，也没使用中文冒号还是这个报错

―

**Reply by Diamondunn** [reply]
所以我该怎么办呢
